crates/bevy_diagnostic/src/log_diagnostics_plugin.rs

```diff
@@ -60,18 +60,18 @@ impl LogDiagnosticsPlugin {
                     diagnostic.name,
                     // Suffix is only used for 's' as in seconds currently,
                     // so we reserve one column for it
-                    format!("{:.6}{:1}", value, diagnostic.suffix),
+                    format_args!("{:.6}{:1}", value, diagnostic.suffix),
                     // Do not reserve one column for the suffix in the average
                     // The ) hugging the value is more aesthetically pleasing
-                    format!("{:.6}{:}", average, diagnostic.suffix),
+                    format_args!("{:.6}{:}", average, diagnostic.suffix),
                     name_width = crate::MAX_DIAGNOSTIC_NAME_WIDTH,
                 );
             } else {
                 info!(
                     target: "bevy diagnostic",
                     "{:<name_width$}: {:>}",
                     diagnostic.name,
-                    format!("{:.6}{:}", value, diagnostic.suffix),
+                    format_args!("{:.6}{:}", value, diagnostic.suffix),
```

Comment:
Can you inline this `format_args!()` directly into the `info!()`?

Reply:
Yes 😄